File: javascript 90 questions/Q3.js


function check_triangle(side1,side2,side3){
    if(side1==side2 && side2==side3){
        console.log("equilateral triangle")
    }else if(side1==side2 || side1==side3 || side2==side3){
        console.log("isoceles triangle")
    }else{
        console.log("scalene triangle")
    }

}

check_triangle(2,2,3);

File: C questions/Q3.c

#include <stdio.h>

int main() {
    int side1 = 30;
    int side2 = 30;
    int side3 = 30;
    if(side1==side2 && side2==side3){
        printf("equilateral triangle");
    }else if(side1==side2 || side1==side3 || side2==side3){
        printf("isoceles triangle");
    }else{
        printf("scalene triangle");
    }
    return 0;
}